src/components/webviews/ReloadInterceptorWebView.js

```diff
@@ -32,8 +32,9 @@ const ReloadInterceptorWebView = React.forwardRef((props, ref) => {
       <ProgressContainer progress={progress}>
         <SupervisedWebView
           {...props}
+          {...(userAgent ? { userAgent } : {})}
```

Comment:
you can remove this line and the `userAgent` declaration on l25 as we don't do any transformation anymore, if provided by the parent it will be injected with the `{...props}` instruction.

Reply:
Also we never inject it from parents on our codebase, so there is no need to make it visible.

Only place userAgent is used is from LauncherView, but it uses the classic WebView.